uat/features/stanford_image_styles.feature

```diff
@@ -1,9 +1,10 @@
+@stanford
 Feature: Stanford Image Styles
   In order to ensure that upgrades do not break existing functionality
   As an administrative user
   I want to ensure that the Stanford Image Styles module is working properly
 
-  @api
+  @api @safe @deploy
```

Comment:
Same comment here as for context_list_active, but I added stanford_image_styles to DRPL-479.